tools/sync-translations.js

```diff
@@ -0,0 +1,98 @@
+/*!
```

Comment:
My preference would be to align this key synchronisation script a lot more with the 3akai-ux synchronisation script. I don't have a strong opinion about whether we use the crowdin JAR file or the node library, but I do think we should pick one that was use in both places to avoid having to maintain 2 completely different approaches. The 3akai-ux synchronisation script also does some things (e.g. removing empty translation files) which this should be doing.

@stuartf : What would be the best approach to aligning these more?